manifests/g/Google/AndroidStudio/Canary/2022.1.1.7/Google.AndroidStudio.Canary.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio.Canary
+PackageVersion: 2022.1.1.7
```

```suggestion
PackageVersion: 2022.1
```
This must match Control Panel.